lib/matplotlib/testing/decorators.py

```diff
@@ -35,58 +35,6 @@ def _cleanup_cm():
         plt.close("all")
 
 
-class CleanupTestCase(unittest.TestCase):
```

Comment:
This isn't a deprecation, you have just removed these. 

What is the motivation for deprecating these?  Just general cleanup?